synapse/config/repository.py

```diff
@@ -70,6 +70,7 @@ def parse_thumbnail_requirements(thumbnail_sizes):
         method = size["method"]
         jpeg_thumbnail = ThumbnailRequirement(width, height, method, "image/jpeg")
         png_thumbnail = ThumbnailRequirement(width, height, method, "image/png")
+        gif_thumbnail = ThumbnailRequirement(width, height, method, "image/gif")
```

Comment:
no idea how to go further than this, as this happens in a `.config.` file, and to "enable" this, you'd need to grab something from the "experimental" config section (as custom for experimental stuff), however, i dunno how to do this without messing up imports or something

Reply:
There are other interdependencies between config sections (see for example `ApiConfig` which checks `self.spaces_enabled`, which is part of `ExperimentalConfig`).  As long as `ContentRepositoryConfig` appears after `ExperimentalConfig` in the `HomeServerConfig` definition, this will work. It's not particularly elegant, but :man_shrugging: